sbws/lib/resultdump.py

```diff
@@ -201,21 +201,25 @@ class Relay:
         Result class to be happy '''
         def __init__(self, fingerprint, nickname, address, master_key_ed25519,
                      average_bandwidth=None, burst_bandwidth=None,
-                     observed_bandwidth=None):
+                     observed_bandwidth=None, bandwidth=None,
```

Comment:
I was about to do that, then I thought that since it's the name stem uses for it and it contains exactly that stem value, might be clearer like this.
Also, at some point I'd like to do not have to redefine these variables in 3 files (see https://trac.torproject.org/projects/tor/ticket/28684#comment:1). Maybe if/when we think about these names more globally, we come out with what works better.

Reply:
Or "cons_bw", if you want to use abbreviations.